subprojects/docs/src/docs/release/notes.md

```diff
@@ -57,6 +57,10 @@ A problem was found with the configuration of task ':test'. Registering invalid
 
 You may now force Gradle to use rich or plain [build output](userguide/console.html#sec:console_build_output) by setting [`org.gradle.console`](userguide/build_environment.html#sec:gradle_configuration_properties) in your `gradle.properties`.
 
+### Specify output directory for source files generated by annotation processor 
+
+When using annotation processors, generated Java source files can be generated into a directory. When the directory is specified via the new `compileJava.options.annotationProcessorGeneratedSourcesDirectory` property, the generated sources are also stored and loaded from the build cache.
```

Comment:
Two recommendations:
"generated Java source files can be generated into a directory" -> "Java source may be generated into a directory specified by the `-s` compiler argument"
"the generated sources are also stored and loaded from the build cache" -> "the generated sources are also stored to and loaded from the build cache"

Reply:
Done.